pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -1112,7 +1112,9 @@ func (c *Cluster) saveMonConfig() error {
 			Monitors: monEndpoints,
 		},
 	}
-	if err := csi.SaveClusterConfig(c.context.Clientset, c.Namespace, c.ClusterInfo, csiConfigEntry); err != nil {
+
+	clusterId := c.Namespace // cluster id is same as cluster namespace for CephClusters
```

Comment:
for rados namespace it can be different....

Reply:
This is the mon code though. The rados namespace code uses a separate clusterID and namespace